vhost-device-console/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 
 ### Added
 
+- Add `--max-queue-size <size>` (`-q <size>` in short) to set the maximum
```

Comment:
Please also add a link to the PR.

See `vhost-device-vsock/CHANGELOG.md` as an example

Reply:
Fixed. My bad, thanks for the kind reminder. I forgot adding an argument is publicly visible.

Reply:
Not a problem at all ;-)

Reply:
The only thing missing for me is the README update, then we should be fine.

Reply:
Fixed :-)